Examples/SwiftExample/Podfile

```diff
@@ -5,12 +5,14 @@ target 'SwiftExample' do
 
   # Pods for SwiftExample
   pod 'Purchases', :path => '../../'
+  pod 'PurchasesCoreSwift', :path => '../../'
```

Comment:
devs won't need to have both listed here, having just `pod 'Purchases'` should suffice, since `PurchasesCoreSwift` will be resolved as the latest pod available.

Reply:
we're listing both just so that we can specify that we want the versions stored locally